packages/sysvars/src/clock.ts

```diff
@@ -0,0 +1,42 @@
+import { assertAccountExists, decodeAccount, type FetchAccountConfig } from '@solana/accounts';
+import { type Codec, getI64Codec, getStructCodec, getU64Codec } from '@solana/codecs';
+import type { GetAccountInfoApi } from '@solana/rpc-api';
+import type { Rpc } from '@solana/rpc-spec';
+import type { Epoch, Slot } from '@solana/rpc-types';
+
+import { fetchEncodedSysvarAccount, SYSVAR_CLOCK_ADDRESS } from './sysvar';
+
+/**
+ * The `Clock` sysvar.
+ *
+ * Information about the network’s clock, ticks, slots, etc.
+ */
+export type SysvarClock = Readonly<{
+    epoch: Epoch;
+    epochStartTimestamp: bigint;
```

Comment:
Yeah probably. I still think the one in. `@solana/rpc-types` will cause confusion, but this is necessary since we're deserializing a `u64` timestamp from state.